src/utils/api.js

```diff
@@ -4,8 +4,16 @@ const server = axios.create({
   baseURL: "https://ajai-back-end-project.onrender.com/api",
 });
 
-export function listOfReviews() {
-  return server.get(`/reviews`).then((response) => {
+export function listOfReviews(search) {
+  let params = {};
+  if (search) {
+    let str = search.split("&");
+    str.map((arr) => {
+      let edited = arr.split("=");
+      params[edited[0]] = edited[1];
+    });
+  }
+  return server.get(`/reviews`, { params }).then((response) => {
```

Comment:
There is an easier way to deal with these 'params' than having to build it yourself, go checkout the nc notes here: [https://notes.northcoders.com/courses/js-front-end/axios#axios-params](https://notes.northcoders.com/courses/js-front-end/axios#axios-params)

This is worth a refactor